guides/v2.3/graphql/reference/customer.md

```diff
@@ -68,6 +68,8 @@ Attribute |  Data Type | Description
 `suffix` | String | A value such as Sr., Jr., or III
 `telephone` | String | The telephone number
 `vat_id` | String | The customer's Tax/VAT number (for corporate customers)
+`custom_attributes` | [CustomerAddressAttribute](#CustomerAddressAttribute) | Address custom attributes
```

Comment:
Place `custom_attributes` and `extension_attributes` in alphabetic order.

Reply:
**done**